### ISSUE TITLE
questions about the number of input images

### ISSUE THREAD

**Issue by vinthony** [iro-cp/FCRN-DepthPrediction#17]
Hi,

thanks for your awesome work!

Could you please tell me how many images are used in the training stage?

I notice that there are many papers just using the labeled data(795 images) to train the model and 654 images to evaluated the metrics.

Is there some rule of the number of training image?

I am looking forward to your reply!

**Comment by iro-cp** [comment]
Hi, thanks!

In the training phase we have extracted ~11-12k images from the raw dataset and then randomly augmented them with small offline transformations (up to approximately 95k). That has been our training set for the provided NYU model. It is also possible to do this with on-the-fly augmentations while training. 

There are indeed methods that use the labeled data only, although some of them work patch-wise. 
Based on some experiments that we ran initially, it's possible to train an architecture with just the labeled data, however more data provide a significant performance gain and generalization capability. This can also be related to the capacity of the network itself (learning more weights might require more samples).

I hope this answers your question :)
